pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "metis-client"
-version = "0.5.0"
+version = "0.5.2"
```

Comment:
Do not touch version numbers, please! Version number is changed automatically by github action.
Use Conventional Commits for commit description, please.
We even have a [guide](https://github.com/metis-science/metis-client/blob/master/CONTRIBUTING.md).